db/data/queries/orderItem.sql

```diff
@@ -1,5 +1,3 @@
-USE `quickOrder`;
```

Comment:
actually i added `USE ` here so that i can execute the script like this `mysql -h localhost -u someuser -p < createTable.sql`

Reply:
I fixed it.